eng/scripts/Language-Settings.ps1

```diff
@@ -152,4 +152,20 @@ function Publish-java-GithubIODocs ($DocLocation, $PublicArtifactLocation)
       }
     }
   }
+}
+
+function Get-java-CSVMetadata () {
+  $metadata = Get-CSVMetadata -MetadataUri $MetadataUri
+  $clientPackages = $metadata | Where-Object { $_.GroupId -eq 'com.azure' } 
+  $nonClientPackages = $metadata | Where-Object { $_.GroupId -ne 'com.azure' -and !$clientPackages.Package.Contains($_.Package) }
+  $uniquePackages = $clientPackages + $nonClientPackages
+  return $uniquePackages
+}
+
+function Get-java-GithubIOArtifact() {
+  return Get-BlobStorage-Artifacts -blobStorageUrl $BlobStorageUrl -blobDirectoryRegex "^java/(.*)/$"
+}
+
+function Generate-java-TocHomepageContent() {
+  return "- name: Azure SDK for Java APIs`r`n  href: api/`r`n  homepage: api/index.md"
```

Comment:
If this is only ever using the language we should just read the language variable instead of having a function for it.

Reply:
I have thought about this. However, in my opionion, we should put as limited things into language-setting as possible.
If we want to pass the `$lang` into the `generateDocfxTocContent`, then we have to move lots of extra common logic into language-setting.ps1.